docs/cluster.md

```diff
@@ -15,20 +15,25 @@ Currently, Jellyfish doesn't offer an option to split a room across multiple mac
 
 ## Configuring a Cluster
```

Comment:
Let's arrange this whole section in the following way:
1. Strategies (here we need to explain the difference between NODES_LIST and DNS strategies)
1. Configuring a Cluster
    1. NODES_LIST
    1. DNS
1. Examples
    1. NODES_LIST
        1. Running from source
        1. Running with Docker (locally)
        1. Running with Docker (globally)
     1. DNS
 1. Verifying that a cluster has been created
 1. Deeper dive into Erlang Distribution